app/res/layout/grid_menu_layout.xml

```diff
@@ -14,9 +14,18 @@
     <FrameLayout
         android:layout_width="match_parent"
         android:layout_height="match_parent"
-        android:layout_alignParentBottom="true"
         android:layout_below="@id/universal_frame_tile">
 
+        <ProgressBar
```

Comment:
since the `RootMenuHomeActivity` uses this layout file as its content view, the holder for the submission progress bar goes here